rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/NodeSourceWindow.java

```diff
@@ -569,6 +570,16 @@ private void replacePolicyItemsInItemList(PluginDescriptor policyPluginDescripto
         resetFormForPolicySelectChange();
     }
 
+    private void validatePolicyNameOrFail(String policyPluginName) {
+        if (this.controller.getModel()
+                           .getSupportedPolicies()
+                           .values()
+                           .stream()
+                           .noneMatch(pluginDescriptor -> pluginDescriptor.getPluginName().equals(policyPluginName))) {
+            throw new IllegalArgumentException("Policy " + policyPluginName + " does not exist");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal " does not exist" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)